### PR TITLE
Adding Footer & Updated navigation

### PR DESCRIPTION
**Pull request by deadlybutter**
This PR

- Adds some simple navigation toggle scripts to `app.js`. I didn't want to use `forge.js` because that requires jquery.
- Adds the footer, basically copy / pasted from prod
- Adds updated navigation markup, profile toggle thing

### CODE REVIEW

**Comment by deadlybutter** [comment]
[(3) As a member, I want to see the same top and footer navigation on Phoenix-ashes and Phoenix-next to have the same links so that I can navigate around the site.](https://trello.com/c/e6Jit9P5/408-3-as-a-member-i-want-to-see-the-same-top-and-footer-navigation-on-phoenix-ashes-and-phoenix-next-to-have-the-same-links-so-that-)

File: resources/assets/app.js [hunk 1]
```diff
@@ -28,10 +28,32 @@ import App from './components/App';
 import { fetchReportbacks } from './actions';
 window.actions = { fetchReportbacks };
 
+function toggleHandler(button, target, toggleClass) {
```
Comment (DFurnes) [inline resources/assets/app.js]: Can you move this into `helpers/index.js`?

File: resources/assets/app.js [hunk 2]
```diff
@@ -28,10 +28,32 @@ import App from './components/App';
...
+  if (!button || !target) return;
+
+  function clickHandler() {
+    console.log('click handler');
```
Comment (DFurnes) [inline resources/assets/app.js]: This doesn't seem like a particularly useful log message. 👾

Comment (deadlybutter) [inline resources/assets/app.js]: whoops

File: resources/assets/app.js [hunk 3]
```diff
@@ -28,10 +28,32 @@ import App from './components/App';
...
+}
+
 ready(() => {
   const appContainer = document.getElementById('app');
 
   if (appContainer) {
     ReactDom.render(<App />, appContainer);
   }
+
+  const navToggle = document.getElementById('js-navigation-toggle');
```
Comment (DFurnes) [inline resources/assets/app.js]: Let's move this into a separate file (say `navigation.js`) that exports an `init` function.

File: resources/views/partials/footer.blade.php [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        </li>
+        <li>
+          <a href="http://www.youtube.com/user/DoSomething1" class="social-icon -youtube" title="dosomething1 on YouTube"><span>dosomething1 on YouTube</span></a>
+        </li>
+      </ul>
+    </div>
+    <div class="footer__column -links">
+        <h4 class="js-toggle-collapsed is-collapsed is-toggleable">Who We Are</h4>
+        <ul>
+          <li><a href="/us/about/who-we-are">What is DoSomething.org?</a></li>
```
Comment (DFurnes) [inline resources/views/partials/footer.blade.php]: Let's prefix each of these with `{{ config('services.phoenix-legacy.url') }}` so they work on next.dosomething.org and review apps.

File: resources/views/partials/navigation.blade.php [hunk 5]
```diff
@@ -0,0 +1,33 @@
+<div class="navigation -floating -white">
+    <a class="navigation__logo" href="{{ url('/') }}"><span>DoSomething.org</span></a>
+    <a  id="js-navigation-toggle" class="navigation__toggle"><span>Show Menu</span></a>
+    <div class="navigation__menu">
+        <ul class="navigation__primary">
+            <li>
+                <a href="{{ url('https://www.dosomething.org/us/campaigns') }}">
```
Comment (DFurnes) [inline resources/views/partials/navigation.blade.php]: Same here, let's make sure this points to the correct environment's pages.

**Comment by deadlybutter** [comment]
All of these suggestions seem fine to me! Will tidy that up post lunch (lunch, not launch. thats 📆 )

**Comment by deadlybutter** [comment]
@DFurnes I think I covered everything, let me know whatcha think!

**Pull request merged**